driver/opentelekomcloud_test.go

```diff
@@ -30,7 +30,7 @@ var (
 		"otc-vpc-name":    vpcName,
 		"otc-tags":        "machine,test",
 	}
-	testEnv = openstack.NewEnv("OTC_")
+	testEnv = openstack.NewEnv("OS_")
```

Comment:
This is used to isolate test environment variables from actually used by the driver